gunicorn/glogging.py

```diff
@@ -328,7 +329,7 @@ def access(self, resp, req, environ, request_time):
             request_time))
 
         try:
-            self.access_log.info(self.cfg.access_log_format, safe_atoms)
+            self.access_log.info(self.cfg.access_log_format, dict(safe_atoms))
```

Comment:
Why are you adding a `dict(…)` wrapper here?

Reply:
@ykusakabe thank you for working on this! We are going to drop Python 2 support pretty soon. See #1764 for details.

Reply:
Yes, you can safely remove that and the other Python 2-related changes (`__future__` imports etc.) in this PR.